text/3817-promote-aarch64-pc-windows-msvc-to-tier-1.md

```diff
@@ -0,0 +1,123 @@
+- Feature Name: promote-aarch64-pc-windows-msvc-to-tier-1
+- Start Date: 2025-05-22
+- RFC PR: [rust-lang/rfcs#3817](https://github.com/rust-lang/rfcs/pull/3817)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+Promote aarch64-pc-windows-msvc to Tier 1 with Host Tools.
+
+# Motivation
+[motivation]: #motivation
+
+About [30% of Rust users use Windows][survey-2024], while the majority of these developers and their
+customers are using x64 hardware, the usage of Arm64 Windows has been growing since it was first
+made available in Windows 10, and has been accelerating, especially with the availability of the
+SnapDragon X processors.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+No changes required: Rust tooling for Arm64 Windows has been available for a while now so this
+doesn't affect the end user experience.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+Tier 1 targets must adhere to the [Tier 1 Target Policy][tier-1-policy]. Going through these
+requirements point-by-point:
+
+> Tier 1 targets must have substantial, widespread interest within the developer community, and must
+> serve the ongoing needs of multiple production users of Rust across multiple organizations or
+> projects. These requirements are subjective, and determined by consensus of the approving teams.
+
+As mentioned above, Windows users comprise a substatial propotion of Rust developers, and Arm64
+hardware is increasingly being used by them and their customers.
+
+> The target maintainer team must include at least 3 developers.
+
+There is an existing [Windows team for Rust][windows-team] and, within Microsoft, Arm64 Windows is
+considered to be a Tier 1 target for internal Rust usage.
+
+> The target must build and pass tests reliably in CI, for all components that Rust's CI considers
+> mandatory.
+> The target must not disable an excessive number of tests or pieces of tests in the testsuite in
```

Comment:
We would like to add `windows-11-arm` runner image to rust-lang/cargo's CI pipeline, to ensure the stability of nightly/unstable features.

(Not a blocker though)

Reply:
Added a note to the Future Possibilities, and an internal Work Item for myself to add it.